test/acceptance/features/bindAppToService.feature

```diff
@@ -344,7 +473,7 @@ Feature: Bind an application to a service
             """
             true
             """
-
+    @olm
```

Comment:
does this scenario depends on the existence of a CSV? If not, we should not introduce this tag

Reply:
I guess it depends on CSV, CSV file define some descriptors. It is using `OLM Operator "backend-new-spec" is running` step.

Reply:
thanks for checking.